src/main/java/gregtech/api/capability/impl/FuelRecipeLogic.java

```diff
@@ -164,15 +214,21 @@ private void tryAcquireNewRecipe() {
                             if (controller.hasMufflerMechanics())
                                 controller.outputRecoveryItems();
 
-                            // increase total on time
+                            // increase total multiblock active time
+                            //TODO, does this correctly account for current maintenance issues scaling the recipe duration in calculateRecipeDuration?
```

Comment:
If the previousRecipe is updated to the modified duration maintenance recipe - then it does properly account for the duration increase. Regular multiblocks might also not account for this properly either - in which case I think it's fine either way as long as they are consistent.